doc/authoring_help.md

```diff
@@ -65,6 +74,72 @@ helps['account'] = """
             """
 </pre>
 
+
+### Example YAML help file, help.yaml (Version 1) ###
+<pre>
+#---------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#---------------------------------------------------------------------------------------------
+
+version: 1
+
+content:
+
+- command:
+    name: account clear
+    summary: Clear account
+    description: Longer summary of how the dummy account clear command works
+    links:
+      - title: Azure Accounts Webpage
+        url: https://azure.microsoft.com/en-us/account/
+      - url: https://aka.ms/just-a-url
+    arguments:
+      - name: --account-name
+        summary: Account name
+        description: |
+          Longer summary with newlines preserved.
+          Preserving newlines is helpful for paragraph breaks.
+        value-sources:
+          - link:
+              title: List accounts
+              command: az account list
+          - link:
+              title: Show an accounts details
+              command: az account show
+          - link:
+              title: Azure Accounts Webpage
+              url: https://azure.microsoft.com/en-us/account/
+          - link:
+              title: Azure Billing Documentation
+              url: https://docs.microsoft.com/en-us/azure/billing/
+          - string:
+              "Account name should be lower case with no numbers or special symbol."
+    examples:
+    - summary: Clear an account
+      description: >
+        This is a longer description of the example.
+        The > character collapses multiple lines into a single line,
+        which is good for on-screen wrapping.
+      command: |
+        az account clear --acount-name myaccount
+
+</pre>
+
+You can also document groups using a similar format.
+
+<pre>
+
+- group:
```

Comment:
Are groups meant to have [examples in help.py](https://github.com/Azure/azure-cli/pull/8325/files#diff-03befe19b92edd4276a02d95696531e3R65)? It seems like they can in knack. But I don't think help.yaml supports this

Reply:
I don't think so, though Jason would want this.